### DISCUSSION TITLE
Minio metrics

### DISCUSSION THREAD

**Discussion by rbs-cloud** [minio/minio#13534] [Q&A]
Hi team

I have enabled the metrics in values file in kubernets environment
Annotation:
    Prometheus.io/scrape: true
    Prometheus.io/path:/ mini/Prometheus/metric s
    Prometheus.io/port: 9000
I am not able to find the s3 request and sts request

Could you please guide us , what needs be configured other than mentioned above

Hi @harshavardhana ,

We are executing sts request through minio and enabled the metrics also but we wanted to check the sts metrics in Prometheus or inside minio container like how many sts request has been sent 

This kind of information whats to check
Could you please guide how to check and that are the metric name
Ex: s3 request i can see the metric like s3_request_total so same like sts also i wanted check

**Comment by eco-minio** [comment]
Are you actually seeing metrics at all?  Your config is incorrect here:
`/ mini/Prometheus/metric s`

Assuming that mini is a typo, and that there is a formatting issue with metrics, prometheus is still case sensitive.  This is also the old endpoint and is not correct, you should be using `/minio/v2/metrics/cluster`.

The following is the list of available metrics:
https://github.com/minio/minio/blob/master/docs/metrics/prometheus/list.md

> Could you please guide us , what needs be configured other than mentioned above

All of this information is included in the documentation, you should check there first, simple configurations like this are already available.  
https://docs.min.io/docs/how-to-monitor-minio-using-prometheus.html

**Reply by rbs-cloud** [reply]
Hi @eco-minio 

Is there any metric that support to STS request(Number of successful and failure STS requests ) metrics like s3(Total number S3 requests) metrics or planning in future if does not support currently.
along with that we need to know below metrics also supported by minio if yes what could be name of the metrics names from below link
https://github.com/minio/minio/blob/master/docs/metrics/prometheus/list.md

1) Time to process each File access request 

2)Volume of data transferred via S3 interface 

3) Number of concurrent requests processed via S3 interface
